### DISCUSSION TITLE
part-ii/2017-69/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#375] [Question comments]
Part II 2017 Galois Theory: Paper 1, Section II, I 

https://questions.tripos.org/part-ii/2017-69/

**Comment by artie2000** [comment]
(a) is bookwork

<details>
<summary>(i)</summary>

$S_4$: the discriminant ($16\cdot101$) is squarefree, and the resolvent cubic ($X^3-8X-4$) is irreducible.

</details>
<details>
<summary>(ii)</summary>

$S_5$:

Irreducible mod 5 by the Artin-Schreier trick (if $\alpha$ is a root, so is $\alpha+1$, so all roots have the same degree).

Factors mod 2 as $(X^2+X+1)(X^3+X^2+1)$.

Applying Dedekind, we have a group of order at least 30 that's not contained in $A_5$.

</details>
